src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -33,25 +36,60 @@ export const SchedulePageModal: FC<SchedulePageModalProps> = ({
 }) => {
   const venueId = useVenueId();
   useConnectRelatedVenues(venueId);
```

Comment:
All of the selectors/etc that are using this could probably just be moved inside this hook to wrap it all up neatly.

Reply:
Semi agree; but I also feel like there's a split between `useConnect`/`useSelector` - so we should create a `useModel` type pattern if we're going to move more this direction..